hack/local-up-cluster.sh

```diff
@@ -1172,7 +1172,7 @@ fi
 if [[ "${START_MODE}" != "nokubelet" ]]; then
   ## TODO remove this check if/when kubelet is supported on darwin
   # Detect the OS name/arch and display appropriate error.
-    case "$(uname -s)" in
```

Comment:
why drop this -s? this is working properly for me locally on OSX

Reply:
Added -s. I was just not sure if that is really needed!